### DISCUSSION TITLE
is there a problem with the snapshot site? (i get ssh check problems)

### DISCUSSION THREAD

**Discussion by jcompagner** [mickaelistria/eclipse-bluesky#1485] [General]
when i want to update to:

Wild Web Developer HTML, CSS, JSON, Yaml, JavaScript, TypeScript, Node tools	1.3.4.202402272001

i get:

An error occurred while collecting items to be installed
session context was:(profile=SDKProfile, phase=org.eclipse.equinox.internal.p2.engine.phases.Collect, operand=, action=).
Problems downloading artifact: osgi.bundle,org.eclipse.wildwebdeveloper,1.1.5.202402272001.
SHA-512 hash is not as expected. Expected: 9e75fc48a51ffa9635c01e72dc02c5f41193c9c8a3efe9a273c4d069240814bf664114734de7cfa9ff514ba3841336d2c4343390340262ad6c3c01674dd0e81e and found d75df3e7cedce2d8b55b90aa3343217fac24c922041310e3f152cc0f2ad32781019b766e052a676251e25fb43847d3993183af1de4c1f0a898779cc8cd46d12e.

anybody else has that problem?

and i like to update, to see where the problem is that i currently have in "ts" editor. 
If i copy/paste then i get extra spaces before it (and the copy really doesn't have that, paste in another editor works fine)

**Comment by jconner2525** [comment]
I am getting the same error.  I even uninstalled Wild Web Developer, restarted Eclipse, and re-installed it from the market place.  Still the same error every time I look for updates.

`An error occurred while collecting items to be installed
session context was:(profile=C_2020-06_eclipse, phase=org.eclipse.equinox.internal.p2.engine.phases.Collect, operand=, action=).
Problems downloading artifact: osgi.bundle,org.eclipse.wildwebdeveloper,1.1.5.202403191205.
SHA-512 hash is not as expected. Expected: 7c03efadebf90142519e299d68a8085b98ad10397376e9772399ddb3263703d7dafe2ebf89d93897b345e9184d2011555ec53336b281c9dd99855f367b6fe9d0 and found 8d9baf17350ef9fbe2f0cc4faec8425f00339bb6cdffdc16cf9d32f178af5abe307e4853f85f15d602254ed0247821b3af588e5684763ad9b5b298de9fe299b3.`

**Reply by mickaelistria** [reply]
I've downloaded https://www.eclipse.org/downloads/download.php?file=/wildwebdeveloper/snapshots/plugins/org.eclipse.wildwebdeveloper_1.1.5.202403191205.jar and then called `sha512sum` on did get the unexpected 8d9baf17350ef9fbe2f0cc4faec8425f00339bb6cdffdc16cf9d32f178af5abe307e4853f85f15d602254ed0247821b3af588e5684763ad9b5b298de9fe299b3 as well, so it might indeed be broken in some way.
Let's convert to an issue.
